### DISCUSSION TITLE
drawio integration on Google Drive: Junk getting left on drawing space.

### DISCUSSION THREAD

**Discussion by PedroSwift** [jgraph/drawio#3710] [Q&A]
As I work, after a short while of moving objects and shifting connectors (process mapping) I get fragments of lines remaining in the display. They can't be selected and are not saved because they disappear when I save, close and re-load.

Anyone else having this issue, couldn't find any posts on this?

![image](https://github.com/jgraph/drawio/assets/7390847/c2bfedb4-8e9f-4e02-b394-13939e1e96d2)

**Comment by davidjgraph** [comment]
https://github.com/jgraph/drawio/issues/3695

**Comment by PedroSwift** [comment]
David, sorry, but I don't see any solution, just others having the same
issue.

I've verified this on Chrome and Firefox.

V21.5.2

Windows 11

I note that I hadn't noticed that the lines weren't captured in my screen
shot, which is interesting.

Please advise.

*PETER ROBERTSON*
p ***@***.***>***@***.***
Peter Robertson Consulting Pty Ltd
*M* +61 407 908 857

**Comment by PedroSwift** [comment]
David, further when I went to the project this morning, the lines were
still present, but offset from the diagram. I hadn't closed the browser,
the machine was on sleep.

[image: image.png]
*PETER ROBERTSON*
p ***@***.***>***@***.***
Peter Robertson Consulting Pty Ltd
*M* +61 407 908 857

**Comment by davidjgraph** [comment]
No, I'm saying this is a duplicate of that ticket. We only track issues in one place.

**Comment by PedroSwift** [comment]
Ah, OK.

*PETER ROBERTSON*
p ***@***.***>***@***.***
Peter Robertson Consulting Pty Ltd
*M* +61 407 908 857
